content/gsoc-2019/contents.lr

```diff
@@ -25,13 +25,23 @@ Here are short bios of each of our 2019 mentors. Please use first names or Slack
 * [Sophine Clachar](https://creativecommons.org/author/sclachar/) (`Sophine` on Slack)
 * [Timid Robot Zehta](https://creativecommons.org/author/timidcreativecommons-org/) (`Timid Robot` on Slack)
 
+### Mentor availability/ Project slots
+
+Google assigns Creative Commons (and all GSoC organizations) a fixed number of project slots. This means that there may be `n` proposals on the project ideas page, but CC may be only be allocated enough project slots to support `< n` projects. The number of project slots allocated to each GSoC organization varies by year. 
+
+You can find more information about the slot allocation process in [the GSoC mentor guide](https://google.github.io/gsocguides/mentor/selecting-students-and-mentors#the-formal-process).
+
 ## Proposal review criteria
 
+Proposal quality is very important, code and community contributions help for different reasons. We are hoping that GSoC will lead to long term contributions to CC, and we see being involved in the community as a sign of general interest in our work. Any code contributions are good for first purposes, contributions to CC repos also show a general interest in our work so they count for both.
```

Comment:
I'd like to edit this for clarity (it was intended to answer a question and not be generally informational) but I'll do it after the PR is merged.